rosa_tutorials/rosa-mobb-aws-load-balancer-operator.adoc

```diff
@@ -0,0 +1,336 @@
+:_content-type: ASSEMBLY
+[id="rosa-mobb-aws-load-balancer-operator"]
+= AWS Load Balancer Operator on ROSA tutorial
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-mobb-aws-load-balancer-operator
+
+toc::[]
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-12
+//---
+//date: '2023-01-03T22:07:08.574151'
+//title: AWS Load Balancer Operator On ROSA
+//aliases: ['/docs/rosa/alb-sts']
+//tags: ["AWS", "ROSA"]
+//authors:
+//  - Shaozhen Ding
+//  - Paul Czarkowski
+//---
+
+include::snippets/mobb-support-statement.adoc[leveloffset=+1]
+
+
+[TIP]
+====
+AWS Load Balancer Operator cannot be used for {product-title} Routes, thus should only be used for individual services or Ingress that does not need the full layer7 capabilties of an {product-title} route.
+====
+
+link:https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/[AWS Load Balancer Controller] is a controller to help manage Elastic Load Balancers for a Kubernetes cluster.
```

Comment:
```suggestion
link:https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/[AWS Load Balancer Controller] is a controller to help manage Elastic Load Balancing v2 (ELBv2) for a Kubernetes cluster.
```

Reply:
Pretty sure this is valid for ELBv2 only? Also, I don't think we use the term Elastic Load Balancers, I think it is Elastic Load Balancing.